buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -870,8 +867,19 @@ class BeamModulePlugin implements Plugin<Project> {
       // Enable errorprone static analysis
       project.apply plugin: 'net.ltgt.errorprone'
 
+      project.dependencies {
+        errorprone("com.google.errorprone:error_prone_core:2.3.1")
+        errorproneJavac("com.google.errorprone:javac:9+181-r4173-1")
```

Comment:
This version looks very "interesting". Can you add a comment so someone who sees it can understand what is going on?

Reply:
Sure, this version is taken directly from EP plugin documentation, support for JDK 8 section to be specific. I'll add comment with short description and link